### PR TITLE
Added respect shield option

### LINKED ISSUE #52: Add a togglable option to not hit when shield is up
**Is your feature request related to a problem? Please describe.**
The mod left clicks (attacks) even if my shield is up. This is annoying since I don't always want the autoclicker to hit while my shield is up.

**Describe the solution you'd like**
A config option "Respect shield" that pauses the attack when shield is up

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by AdyTech99**
Closes #52

### CODE REVIEW

File: src/main/java/pro/mikey/autoclicker/AutoClicker.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package pro.mikey.autoclicker;
 
+import ca.weblite.objc.Client;
```
Comment (MichaelHillcox) [inline src/main/java/pro/mikey/autoclicker/AutoClicker.java:3]: Client?

**Review by MichaelHillcox — CHANGES_REQUESTED** [review]
Just one issue, otherwise, nice work :D

File: src/main/java/pro/mikey/autoclicker/AutoClicker.java (hunk 1 above)
Comment (AdyTech99) [reply src/main/java/pro/mikey/autoclicker/AutoClicker.java]: Fixed lol. Tried to type MinecraftClient it ended up importing Client.

**Comment by AdyTech99** [comment]
Ding

**Pull request merged**